### PR TITLE
chore(schemaspy): add schemaspy image with mysql & postgresql connectors

### PR DESCRIPTION
**Pull request by bmaziere**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ matrix:
     - env: LANGUAGE=react-native VERSION=1
     - env: LANGUAGE=ruby         VERSION=2.6
     - env: LANGUAGE=sonar        VERSION=3.3      GLIBC_VERSION=2.27-r0 SONARSCANNER_VERSION=3.3.0.1492
+    - env: LANGUAGE=schemaspy    VERSION=6        SCHEMASPY_VERSION=6.0.0 POSTGRESQL_CONNECTOR_VERSION=9.4.1212 MYSQL_CONNECTOR_VERSION=8.0.15
```
Comment (mremi) [inline .travis.yml]: please sort by language and sort the given vars

File: schemaspy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+ADD open-sans.tar.gz /usr/share/fonts/
+
+RUN apk add --update curl graphviz \
+    && fc-cache -fv \
+    && echo "Install Mysql connector" \
+    && curl -sSL https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.tar.gz | tar -xz \
+    && ln -s mysql-connector-java-${MYSQL_CONNECTOR_VERSION}/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.jar mysql-connector-java.jar \
+    && echo "Install Postgresql connector" \
+    && curl -Ls "https://jdbc.postgresql.org/download/postgresql-${POSTGRESQL_CONNECTOR_VERSION}.jar" -o "postgresql-connector-java.jar" \
```
Comment (mremi) [inline schemaspy/Dockerfile]: `-sSL`

File: travis.py [hunk 3]
```diff
@@ -183,6 +183,9 @@ def run_build(buildInfo):
         if language == "sonar":
             build_args = "%s --build-arg GLIBC_VERSION=%s --build-arg SONARSCANNER_VERSION=%s" % (build_args, os.environ.get("GLIBC_VERSION"), os.environ.get("SONARSCANNER_VERSION"))
 
+        if language == "schemaspy":
+            build_args = "%s --build-arg SCHEMASPY_VERSION=%s --build-arg MYSQL_CONNECTOR_VERSION=%s --build-arg POSTGRESQL_CONNECTOR_VERSION=%s" % (build_args, os.environ.get("SCHEMASPY_VERSION"), os.environ.get("MYSQL_CONNECTOR_VERSION"), os.environ.get("POSTGRESQL_CONNECTOR_VERSION"))
```
Comment (mremi) [inline travis.py]: please sort args

**Comment by mremi** [comment]
@Tetradeus WDYT?

File: schemaspy/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
+FROM java:8u111-jre-alpine
```
Comment (fpavageau) [inline schemaspy/Dockerfile]: We should probably use `adoptopenjdk/openjdk8:alpine-jre` instead, or `adoptopenjdk/openjdk8:jre8u212-b03-alpine` if we want to be super strict, instead. The "official"  Java / OpenJDK images are a bit shady, they're based on the Debian packages which do not follow the real OpenJDK releases (see [this mail](https://mail.openjdk.java.net/pipermail/jdk8u-dev/2019-May/009330.html) for example), and nobody's really sure who's providing them.

**Comment by bmaziere** [comment]
thx for review @fpavageau.
A schemaspy/schemaspy image already exists so we will use it directly inside projects. That means less image to maintain here. Closing this one then

**Pull request closed without merging**